openpype/hosts/unreal/api/plugin.py

```diff
@@ -1,7 +1,242 @@
 # -*- coding: utf-8 -*-
+import ast
+import collections
+import sys
+import six
 from abc import ABC
 
-from openpype.pipeline import LoaderPlugin
+import unreal
+
+from .pipeline import (
+    create_publish_instance,
+    imprint,
+    lsinst,
+    UNREAL_VERSION
+)
+from openpype.lib import (
+    BoolDef,
+    UILabelDef
+)
+from openpype.pipeline import (
+    Creator,
+    LoaderPlugin,
+    CreatorError,
+    CreatedInstance
+)
+
+
+class UnrealBaseCreator(Creator):
```

Comment:
I guess it has to be still declared as abstract, because it is missing implementation of abstract property `family`